packages/ui-components/tsconfig.lib.json

```diff
@@ -9,7 +9,8 @@
     ],
     "paths": {
       "@/*": ["./src/*"],
-      "@openops/shared": ["../../packages/shared/src"]
+      "@openops/shared": ["../../packages/shared/src"],
+      "@openops/blocks-framework": ["../../packages/blocks/framework/src"]
```

Comment:
missing dependency, types were showing as error in `/packages/ui-components/src/lib/types.ts`